### DISCUSSION TITLE
Inverting Artist & Album in the Playlist view ?

### DISCUSSION THREAD

**Discussion by KorenLesthe** [TT-ReBORN/Georgia-ReBORN#178] [Q&A] [answered]
Hi !
I just installed the 3.0 RC2 of Georgia-REBORN and I'm loving it so far.
I was wondering if it is possible to invert the way Artists and Albums are displayed in the playlist ; preferring to see the Album first and the artist(s) name(s) under it. So far, I've searched for every option in the various menus of this amazing theme but I haven't found a way.

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @KorenLesthe,

this feature was missing but I have implemented it in the latest commit https://github.com/TT-ReBORN/Georgia-ReBORN/commit/5be39d6ff485e9eca805bee531c4fa3dc98cc7ba:
```
Added option to flip playlist header rows
- Top menu > Options > Playlist > Album header > Flip header rows
or
- Playlist context menu > Appearance > Headers > Flip header rows
```

You need to download the DEV version and replace your files to be able to use it:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip

---

There is also the option to flip the artist and album labels in the Library when using the album art grid view:
`Top menu` > `Options` > `Library` > `Album art` > `Labels` > `Flip`
or
`Right click Library content menu` > `Library options menu` > `Album art` > `Labels` > `Flip`.

-TT

**Reply by KorenLesthe** [reply]
Thank you !
